packages/web/stores/imperator-queries/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./pools";
```

Comment:
First, I put it in `web` package because I thought it would only be used in `web` package and I put in `stores` folder because other query things are in `store` package.